tests/fixtures/data.yml

```diff
@@ -9,12 +9,16 @@ person:
     ORIGINAL_HIRE_DATE: !!timestamp 2001-01-01 00:00:00.0
     APPOINTMENT_END_DATE: !!timestamp 2010-01-01 00:00:00.0
     JOB_TITLE: Adjunct Professor
-    PERSONNEL_SUBAREA_CODE: CFAC
-    HR_ORG_UNIT_TITLE: Chemisty
+    PERSONNEL_SUBAREA_CODE: CFAT
+    HR_ORG_UNIT_ID: "1"
+    HR_ORG_LEVEL4_NAME: Chemistry
```

Comment:
Hmmm. I'm not sure why I needed to add this back to the `person` then. The app runs against oracle by pulling that data from the `dlc` table. I'll look into this further.